mithril-aggregator/src/message_adapters/to_cardano_stake_distribution_list_message.rs

```diff
@@ -0,0 +1,55 @@
+use mithril_common::entities::{CardanoStakeDistribution, SignedEntity};
+use mithril_common::messages::{
+    CardanoStakeDistributionListItemMessage, CardanoStakeDistributionListMessage, ToMessageAdapter,
+};
+
+/// Adapter to convert a list of [CardanoStakeDistribution] to [CardanoStakeDistributionListMessage] instances
+#[allow(dead_code)]
+pub struct ToCardanoStakeDistributionListMessageAdapter;
+
+impl
+    ToMessageAdapter<
+        Vec<SignedEntity<CardanoStakeDistribution>>,
+        CardanoStakeDistributionListMessage,
+    > for ToCardanoStakeDistributionListMessageAdapter
+{
+    /// Method to trigger the conversion
+    fn adapt(
+        snapshots: Vec<SignedEntity<CardanoStakeDistribution>>,
+    ) -> CardanoStakeDistributionListMessage {
+        snapshots
+            .into_iter()
+            .map(|entity| CardanoStakeDistributionListItemMessage {
+                epoch: entity.artifact.epoch,
```

Comment:
Same comment for the epoch used.

Reply:
Fixed in this commit: https://github.com/input-output-hk/mithril/commit/3b440ded30f374e0d3d74603047961aa919c92ab